primedev/shared/exploit_fixes/exploitfixes.cpp

```diff
@@ -120,19 +120,30 @@ bool, __fastcall, (void* pMsg)) // 48 8B D1 48 8B 49 18 48 8B 01 48 FF 60 10
 			if (!nameValid || !valValid)
 				return BLOCKED_INFO("Missing null terminators");
 
-			ConVar* pVar = g_pCVar->FindVar(entry->name);
-
-			if (pVar)
+			// we only need to check if these cvars are valid on client as it will set actual cvars there
+			// on server this won't set any actual convars, only keyvalues in the player, which doesn't have really any potential for dumb
+			// stuff
+			if (!bIsServerFrame)
 			{
-				memcpy(
-					entry->name,
-					pVar->m_ConCommandBase.m_pszName,
-					strlen(pVar->m_ConCommandBase.m_pszName) + 1); // Force name to match case
-
-				int iFlags = bIsServerFrame ? FCVAR_USERINFO : FCVAR_REPLICATED;
-				if (!pVar->IsFlagSet(iFlags))
-					return BLOCKED_INFO(
-						"Invalid flags (" << std::hex << "0x" << pVar->m_ConCommandBase.m_nFlags << "), var is " << entry->name);
+				ConVar* pVar = g_pCVar->FindVar(entry->name);
+				if (pVar)
+				{
+					memcpy(
+						entry->name,
+						pVar->m_ConCommandBase.m_pszName,
+						strlen(pVar->m_ConCommandBase.m_pszName) + 1); // Force name to match case
+
+					if (!pVar->IsFlagSet(FCVAR_REPLICATED))
+					{
+						return BLOCKED_INFO(
```

Comment:
unsure when exactly this was made to return but it shouldn't! with this return in place, i believe this pr does not fix the issue it is attempting to fix

Reply:
in my original change this was just an spdlog::warn without a return, not return BLOCKED_INFO

Reply:
here is a fix for that https://github.com/R2Northstar/NorthstarLauncher/pull/683

Reply:
Fixed via e334f49f28c70fa9a309de9d7dbf0f9e7008aaad